app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):

```

Comment:
`Dwarf` class does not implement all abstract methods from the `Player` class, inherit it from `ABC` to make abstract as well.